exercises/stack_01.js

```diff
@@ -0,0 +1,82 @@
+window.Collections = {};
```

Comment:
You'll often see this line written this way:

```
window.Collections =  window.Collections || {};
```

Which basically says, "Assign that global to the empty object only if it doesn't already exist." This stops you from "clobbering" a module during a dynamic import. Ugh, old-school JavaScript's modules really are annoying.